cmd/wallet-web/src/pages/Store.vue

```diff
@@ -248,6 +247,10 @@ export default {
     fetchAllVaults: async function (token, collectionManager) {
       const { contents } = await collectionManager.getAll(token);
       this.vaults = Object.values(contents).map((vault) => vault);
+      // Default vault is selected vault by default, it is created on vault setup and must be only one.
+      const defaultVault = this.vaults.filter((vault) => vault.name.includes('Default'));
+      const fetchedDefaultId = defaultVault.map((value) => value.id);
+      this.selectedVault = fetchedDefaultId.values().next().value;
```

Comment:
In addition, since you already have a method to set `this.selectedVault`, you should use it here as well instead of duplicating your code:
```js
this.setSelectedVault(defaultVaultId);
```